oioswift/proxy/controllers/obj.py

```diff
@@ -280,7 +280,8 @@ def get_object_head_resp(self, req):
                 nb_chunks_ok = 0
                 for entry in entries[1]:
                     try:
-                        storage.blob_client.chunk_head(entry['url'])
+                        storage.blob_client.chunk_head(
+                            entry['url'], headers=oio_headers)
```

Comment:
This looks like a good place for calling `chunk_head_many()` (not implemented yet).